packages/Polyfill/src/FeatureSupport/FunctionSupportResolver.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Polyfill\FeatureSupport;
+
+use Rector\Php\PhpVersionProvider;
+
+final class FunctionSupportResolver
+{
+    /**
+     * @var string[][]
+     */
+    private const FUNCTIONS_BY_VERSION = [
+        '5.6' => ['session_abort'],
+    ];
```

Comment:
@staabm Here is just the code example you showed me. I need your help to complete list as far as possible.

Reply:
I need to think about how to compile such method most easily

Reply:
After thinking about it I feel we could compile this list from exisiting polyfil libs, e.g.
https://github.com/symfony/polyfill-php56
https://github.com/symfony/polyfill-php70
https://github.com/symfony/polyfill-php71
https://github.com/symfony/polyfill-php72
https://github.com/symfony/polyfill-php73
https://github.com/symfony/polyfill-php74

Reply:
Another thing I checked via github code-search
https://github.com/search?l=PHP&q=function_exists&type=Code

There we can see most examples are like
```
if( ! function_exists( 'supernova_comment' ) ){
function supernova_comment($comment, $args, $depth) {
 // impl
}
```

In these cases we could eleminate the code when we are sure that the function beeing checked is one from the php standard library

Reply:
> After thinking about it I feel we could compile this list from exisiting polyfil libs, e.g.

Great!

> Another thing I checked via github code-search

Try to keep issues separated, so we don't get stuck with complexity. This is another use case.

Reply:
> This is another use case.

Absolutely. Just wanted to let you know which things I tried

Reply:
I see

Reply:
There are many new PRs merged every day so there are new conflicts to rebase on.
I'll merge this now to prevent that.

You can send the function list as a new PR targeting this class :+1:

Reply:
@staabm Any update on this? The rule is pretty lonely with just 1 function now :D

Reply:
Was on vacation till yesterday.. will need a few more days to catch up

Reply:
here we go https://github.com/rectorphp/rector/pull/2673